Environment_Engine/Create/Location.cs

```diff
@@ -47,6 +47,7 @@ public static partial class Create
         [Input("utcOffset", "The offset from UTC for the location (positive or negative), default 0")]
         [Input("name", "The name of the location, default empty string")]
         [Output("location", "An Environment location object - used for defining locations in space for climate analysis")]
+        [Deprecated("3.0", "Deprecated in favour of default create components produced by BHoM")]
```

Comment:
Hmm on reflection, could we add a new query method which checks if a `Location` object is valid which runs the `if` checks that were included in this create method?